project/Dependencies.scala

```diff
@@ -26,4 +26,9 @@ object Dependencies {
 
   val play = mainPlay ++ testPlay
 
+  def pickVersion(scalaVersion: String, ver211: String, ver212: String) = CrossVersion.partialVersion(scalaVersion) match {
```

Comment:
Sbt has mechanism for that: http://www.scala-sbt.org/0.12.2/docs/Detailed-Topics/Cross-Build.html